projects/fsastorefrontlib/src/core/checkout/services/checkout-config.service.ts

```diff
@@ -114,6 +114,17 @@ export class FSCheckoutConfigService extends CheckoutConfigService {
       : null;
   }
 
+  getInitialStepForCategory(category: string): FSCheckoutStep {
+    let initialStep: FSCheckoutStep;
+    for (const step of this.fsCheckoutConfig.checkout.steps) {
+      if (!(<FSCheckoutStep>step).restrictedCategories.includes(category)) {
+        initialStep = step;
+        break;
```

Comment:
`break` should be used only in `while` and `switch` statements. Is it really needed here?

Reply:
Why break can't be used in a for loop? I think it could be used in any type of a loop :) 
But I agree maybe there is a prettier way with map or filter to do this.

Reply:
You are right, but I've said it shouldn't be used in a sense that perhaps it would be better to use it with some newer es6 method. In this case it's recommended to perhaps use `find()` method.
```
initialStep = this.fsCheckoutConfig.checkout.steps.find(
  step => !(<FSCheckoutStep>step).restrictedCategories.includes(category)
);
```
Or something like that....

Reply:
Switched to ES6 native find method :)